doc/pages/administrator/062-handling_certificates.md

```diff
@@ -0,0 +1,114 @@
+# Handling Astarte certificates
+
+Astarte heavily requires SSL in a number of interactions, even though this can be bypassed with
+`ssl: false`. In general there are two main scenarios when dealing with certificates: either you
+already purchased SSL certificates for your domains or you might want the task to be handled by
+Let's Encrypt through cert-manager.
+
+The two alternative procedures for securing your Astarte deployment are outlined in the following
+sections.
+
+## Use your own certificates
+
+If you already own certificates for your domains, all it's needed is creating a TLS secret in the
+namespace in which Astarte resides. Assuming that the certificate and key are saved respectively as
+`cert.pem` and `privkey.pem`, simply run:
+
+```bash
+$ kubectl create secret tls astarte-tls-cert -n astarte \
+  --cert=cert.pem --key=privkey.pem
+```
+
+## Use Let's Encrypt certificates with cert-manager
+
+The current section describes the procedure for setting up a `ClusterIssuer` to use Google CloudDNS
```

Comment:
People reading this doc for the first time might not now what's a cluster issuer and how it relates to Astarte. I'd add a one-liner intro like
```
The process of obtaining a tls certificate from Let's Encrypt is handled by cert manager using a cluster issuer.
```
From there you can continue with something along the lines of
```
The issuer will query the Let's Encrypt API and handle the challenge to confirm you're the right owner of the specified domain.
We support two types of challenges, DNS and HTTP [...]
```